src/app/routes/activation.ts

```diff
@@ -16,16 +16,28 @@ class ActivationController {
     this.service = service;
   }
 
+  async sendDeactivationEmail(req: Request<{ email: string }>, res: Response) {
+    const email = req.params.email.toLowerCase();
+    const user = await this.service.User.FindUserByEmail(email);
```

Comment:
Seems like the found user is only for analytics. Is better to do this after handling the main action that is sending the deactivation email. This reduces latency as the user query + track can be deferred.

Otherwise, check if the user is not null or return a 404 as the deactivate call line will throw an error with a 500 code status if the user is not found.